README.md

```diff
@@ -215,6 +215,18 @@ The number of cache misses that have occurred on the node. A cache miss occurs w
 #### graph_memory_usage
 The current weight of the cache (the total size in native memory of all of the graphs) in Kilobytes.
 
+#### graph_index_requests
+The number of requests to add the knn_vector field of a document into a graph.
+
+#### graph_index_errors
+The number of requests to add the knn_vector field of a document into a graph that have produced an error.
+
+#### graph_query_requests
+The number of graph queries that have been made. 
+
+#### graph_query_errors
```

Comment:
How about we add metrics for counting KNNQueries?

Reply:
Makes sense. Will add